.gitignore

```diff
@@ -20,6 +20,9 @@ logs/modperl_error_log
 
 # Local databases
 data/mongodb
+Lang.openfoodfacts.*
```

Comment:
Is `openfoodfacts` static or could it also be `openpetfoodfacts`?

Reply:
I have put `Lang.open*`.

I think I will open a new bug to suggest moving this files from root directory.